includes/ConfigDefaults.php

```diff
@@ -226,6 +226,12 @@
     // see crypto_app.js for constants in the range crypto_key_version_constants
     // Generally higher is newer + better.
     'encryption_key_version_new_files' => 1,
+
+    // for details of possible values see crypto_password_version_constants
+    // in the file js/crypter/crypto_app.js
+    'encryption_random_password_version_new_files' => 2,
+    
+    'encryption_password_hash_iterations_new_files' => 150000,
```

Comment:
Suggest to reference the .md for an explanation, and/or warn against frivolously editing this value.  Again, this value is quite low, almost invalid at the present date (safe until 2020).  Passwords are really, really rotten for anything but the simplest form of security and should not be considered by any secure usage pattern, and it should not be a default, or if it is then at least at a secure level by default.  Security until 2020 is rediculously low.

Reply:
I added (once CI passes) the reference with the introduction of https://github.com/filesender/filesender/pull/649. I think the https://docs.filesender.org/v2.0/admin/configuration/#encryption_password_hash_iterations_new_files link is where most admins will be looking to start with, but if it helps somebody who found this value directly then it is well worth putting the link in there.